packages/app/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {},
   "devDependencies": {
     "@cypress-design/vue-icon": "^0.12.1",
+    "@cypress-design/vue-statusicon": "0.1.4",
```

Comment:
Not really your wheelhouse, but I wonder why they don't just publish a single package, `cypress-design/core`, with everything we need.

Reply:
@lmiller1990 That is because of versioning.
Upgrading a package should not risk breaking the rest of the platform.
It is a way of minimizing the surface risk while still mutualizing the components.

We might create a `@cypress-design/core` at some point once it stabilizes.